lib/resourcemerge/admissionregistration_test.go

```diff
@@ -0,0 +1,644 @@
+package resourcemerge
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/utils/pointer"
```

Comment:
nit: no reason to keep these separate?

Reply:
Nice catch! I must have deleted the line and forgotten about it.

Reply:
Fixed the redundant line breaking in `lib/resourcemerge/admissionregistration_test.go` and `lib/resourceapply/admissionregistration.go`.